xml/ha_concepts.xml

```diff
@@ -607,122 +607,99 @@
+     </para>
     <variablelist>
-     <varlistentry xml:id="vle.crm">
-      <term>Cluster Resource Manager (CRM)</term>
-      <listitem>
-       <para>
-        Every action taken in the resource allocation layer passes through
-        the Cluster Resource Manager. If other components of the resource
-        allocation layer (or components which are in a higher layer) need to
-        communicate, they do so through the local CRM. On every node, the
-        CRM maintains the
-        <xref linkend="vle.cib" xrefstyle="select:nopage"/>.
-       </para>
-      </listitem>
-     </varlistentry>
-     <varlistentry xml:id="vle.cib">
-      <term>Cluster Information Base (CIB)</term>
+     <varlistentry xml:id="vle.lrm">
+      <term>Local Resource Manager</term>
       <listitem>
-       <para>
-        The Cluster Information Base is an in-memory XML representation of
-        the entire cluster configuration and current status. It contains
-        definitions of all cluster options, nodes, resources, constraints
-        and the relationship to each other. The CIB also synchronizes
-        updates to all cluster nodes. There is one master CIB in the
-        cluster, maintained by the
-        <xref linkend="vle.dc" xrefstyle="select:nopage"/>. All other nodes
-        contain a CIB replica.
-       </para>
+      <para>
+        The local resource manager is located between the Pacemaker layer and the
+        resources layer on each node. It is implemented as <systemitem
+        class="daemon">pacemaker-execd</systemitem> daemon. Through this daemon,
+        Pacemaker can start, stop  and monitor resources or fence nodes.
+      </para>
       </listitem>
      </varlistentry>
+      <varlistentry  xml:id="vle.cib">
+       <term>Cluster Information Database (CIB)</term>
+       <listitem>
+        <para>
+         On every node, Pacemaker maintains the cluster information database
+         (CIB). It is an XML representation of the cluster configuration
+         (including cluster options, nodes, resources, constraints and the
+         relationship to each other). The CIB also reflects the current cluster
+         status. There is one master CIB in the cluster, hosted on the node that
+         has been elected as Designated Coordinator (DC). From the DC, the CIB is
+         synchronized to the other cluster nodes, which contain a CIB replica
+         each. The <systemitem class="daemon">pacemaker-based</systemitem>
+         daemon takes care of reading and writing cluster configuration and
+         status.</para>
+       </listitem>
+      </varlistentry>
      <varlistentry xml:id="vle.dc">
       <term>Designated Coordinator (DC)</term>
       <listitem>
        <para>
-        One CRM in the cluster is elected as DC. The DC is the only entity
+        The DC is elected from all nodes in the cluster after a membership change
+        (nodes joining or leaving the cluster). The DC is the only entity
```

Comment:
Election of DC doesn't always happen after a membership change :-) Only if there's no DC yet or the DC failed or left the cluster.

Reply:
I adjusted the sentence.